.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*.c]
```

Comment:
For space indent, yes.
For tab indent it does not matter, everyone has their own preference?
(tabs are tabs)

I have [*.{c,cpp,h,hpp}] now, any more extensions I should add?